src/components/ninja.js

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+
+function Quote() {
+  const [quote, setQuote] = useState([]);
```

Comment:
- [ ] It would be better if we set states for `loading` and `error` state. This works well when working with API's, because the API calls can take some time to return data so whilst waiting, we will trigger the loading state and when there is any error, the error state gets to print an error message or something
![Screenshot from 2023-07-07 22-32-33](https://github.com/HFG43/math_magicians/assets/46388113/434e24fb-ee09-472f-838a-865764f10f53)
For example; `const [hasError, setHasError] = useState(false);
  const [isLoading, setIsLoading] = useState(false)
`